week-9/Homework/mandatory/1-practice/2-code-reading.md

```diff
@@ -33,6 +34,8 @@ console.log(y)
 ```
 
 What will be the output of this code. Explain your answer in 50 words or less.
+The first output will be 10, because in the console.log on line 32 we call the function witch will execute a console.log.
```

Comment:
The variable `x` here is defined in global scope, so you can call `console.log` on the variable within the function.